schemes/rrtmgp/rrtmgp_sw_mcica_subcol_gen.meta

```diff
@@ -0,0 +1,122 @@
+[ccpp-table-properties]
+  name = rrtmgp_sw_mcica_subcol_gen
+  type = scheme
+  dependencies = ../../../../../share/RandNum/src/shr_RandNum_mod.F90
```

Comment:
Given that this is a CAM-SIMA specific source file I wouldn't include it here as a dependency (instead we should replace it at some point with a different, more portable utility scheme).